bankv2.rb

```diff
@@ -64,26 +57,32 @@ def add_owner
     end
   end
 end
-  # class Owner
-  #
-  #   attr_reader :first_name, :last_name, :street_1, :street_2, :city, :state, :zip, :ssn
-  #
-  #   def initialize(owner_hash)
-  #     @first_name = owner_hash[:first_name]
-  #     @last_name = owner_hash[:last_name]
-  #     @street_1 = owner_hash[:street_1]
-  #     @street_2 = owner_hash[:street_2]
-  #     @city = owner_hash[:city]
-  #     @state = owner_hash[:state]
-  #     @zip = owner_hash[:zip]
-  #     @ssn = owner_hash[:ssn]
-  #   end
-    # def print_info
-    #   puts "#{@first_name} #{@last_name} is the owner of this account, from #{@city}, #{@state}."
-    # end
-  # end
 
-# name = Bank::Owner.new({name: "Sarah", zip: "98933"})
+#   class Owner
```

Comment:
Since you moved the `Owner` code to it's own file, I would delete this commented out section